BootloaderCorePkg/Tools/CfgDataStitch.py

```diff
@@ -69,16 +69,8 @@ def cfgdata_stitch(ifwi_file, ifwi_out_file, cfg_dir, key_dir, script_dir, tool_
         else:
             script_dir = get_script_dir()
 
-    if key_dir == '':
-        if fv_dir:
-            key_dir = os.path.realpath(os.path.join(fv_dir, '../../../../BootloaderCorePkg/Tools/Keys'))
-        else:
-            key_dir = get_script_dir()
-
-    if os.path.isdir(key_dir):
-        key_file = os.path.join(key_dir, 'TestSigningPrivateKey.pem')
-    else:
-        key_file = key_dir
+    if key_file == '':
+        raise Exception("Key file is not specified!!")

```

Comment:
Can't we use CFGDATA_KEY by default here....if one is not provided ?

Reply:
We are planning to provide KEY_ID or complete key path for standalone scripts.
I think we could default CFGDATA_KEY at later stage.